aten/src/ATen/core/type.cpp

```diff
@@ -143,6 +143,11 @@ std::ostream& operator<<(std::ostream & out, const Type & t) {
   return out;
 }
 
+std::ostream& operator<<(std::ostream& os, const SymInt& s) {
```

Comment:
There's no reason to take this by reference since this is an int. Just take a SymInt by value